packages/react-app/src/NftyWallet.js

```diff
@@ -95,427 +93,193 @@ export default function NftyWallet(props) {
-              ink={ink}
-              ipfsConfig={ipfsConfig}
-              ipfsConfigInfura={ipfsConfigInfura}
-              ipfsHash={ipfsHash}
-              key={renderKey}
-              kovanProvider={props.kovanProvider}
-              mainnetProvider={props.mainnetProvider}
-              metaProvider={props.metaProvider}
-              readContracts={props.readContracts}
-              readKovanContracts={props.readKovanContracts}
-              setArtist={setArtist}
-              setDrawing={setDrawing}
-              setInk={setInk}
-              setIpfsHash={setIpfsHash}
-              setMode={setMode}
-              setTab={setTab}
-              setViewDrawing={setViewDrawing}
-              transactionConfig={transactionConfig}
-              upgradePrice={upgradePrice}
-              viewDrawing={viewDrawing}
+          <Switch>
+            <Route exact path="/">
+              <Homepage
+                localProvider={props.localProvider}
+                injectedProvider={props.injectedProvider}
+              />
+            </Route>
+            <Route path="/debug">
+              <DebugContracts {...props} />
+            </Route>
+            <Route path="/recent">
+              <AllInks />
+            </Route>
+            {/* <Route path="/curated">
+            <CuratedInks {...props} />
+          </Route> */}
+
+            <Route path="/holdings">
+              <Holdings
+                {...props}
+                address={props.address}
+                transactionConfig={transactionConfig}
+                upgradePrice={upgradePrice}
+              />
+            </Route>
+
+            <Route path="/artist/:address">
+              <Artist {...props} />
+            </Route>
+
+            <Route path="/create-art">
+              <CreateFile
+                {...props}
+                key={renderKey}
+                canvasKey={canvasKey}
+                address={props.address}
+                mainnetProvider={props.mainnetProvider}
+                injectedProvider={props.injectedProvider}
+                metaProvider={props.metaProvider}
+                kovanProvider={props.kovanProvider}
+                readKovanContracts={props.readKovanContracts}
+                mode={mode}
+                ink={ink}
+                ipfsHash={ipfsHash}
+                setMode={setMode}
+                setIpfsHash={setIpfsHash}
+                setInk={setInk}
+                drawing={drawing}
+                setDrawing={setDrawing}
+                viewDrawing={viewDrawing}
+                setViewDrawing={setViewDrawing}
+                ipfsConfig={ipfsConfig}
+                ipfsConfigInfura={ipfsConfigInfura}
+                gasPrice={props.gasPrice}
+                calculatedVmin={calculatedVmin}
+                transactionConfig={transactionConfig}
+              />
+            </Route>
+
+            {/* <Route path="/create">
+            <div>
+              <CreateInk
```

Comment:
Is this route and component still needed? If not, we should delete it to make sure it doesn't compile.

Reply:
Yeap. We do not need it. We left it for debugging purposes only.